marketing/webinars: document how to notify users of webinars on customers' instances (#607)
sourcegraph/about: handbook/marketing/webinars.md

- Registration landing page form
- Thanks for registering email which includes the webinar link

### Announcement on customer's Sourcegraph instance

Ask the customer:

> > Could you add the following configuration to your Sourcegraph instance's global user settings found at https://sourcegraph.example.com/site-admin/global-settings ? This will let all the users visiting Sourcegraph know that they can join the webinar via a dismissible notice:

Reviewer: ```suggestion
> Could you add the following configuration to your Sourcegraph instance's global user settings found at https://sourcegraph.example.com/site-admin/global-settings? This will let all the users visiting Sourcegraph know that they can join the webinar via a dismissible notice:
```

>
> ![image](https://user-images.githubusercontent.com/3173176/75200913-423db180-5724-11ea-9ee9-fbea5be2129b.png)
>
> ```json
>  "notices": [
>    {
>      "dismissible": true,
>      "location": "top",

> >      "message": "Join us and the Sourcegraph team for the live Zoom webinar on [Tuesday, Feb 25, 2020 @ 3pm Central](https://sourcegraph.com) to learn how Sourcegraph can improve your workflow!"

Reviewer: ```suggestion
>      "message": "Join us and the Sourcegraph team for the live Zoom webinar on [Tuesday, Feb 25, 2020 @ 3pm Central](https://sourcegraph.com) to learn about Universal Code Search and improving your workflow!"
```

>      }
>  ],
> ```

> **IMPORTANT**: Before sending the above email, be sure to replace the date with the actual date, and the link `https://sourcegraph.com` with the actual Zoom webinar link!

Reviewer: ```suggestion
**IMPORTANT**: Before sending the above email, make sure you update the webinar date and URL!
```
